PostHogTests/TestUtils/MockPostHogServer.swift

```diff
@@ -53,6 +54,10 @@ class MockPostHogServer {
                 self.replayVariantName: self.replayVariantValue,
             ]
 
+            if self.quotaLimitFeatureFlags {
```

Comment:
@dmarticus can we add a unit test for this as well?
that would go here https://github.com/PostHog/posthog-ios/blob/main/PostHogTests/PostHogFeatureFlagsTest.swift

Reply:
done!